src/p_mobj.h

```diff
@@ -296,6 +297,14 @@ typedef struct mobj_s
 	fixed_t old_spritexscale, old_spriteyscale;
 	fixed_t old_spritexoffset, old_spriteyoffset;
 
+	// saltyhop! hardcode edition
```

Comment:
I think it would be better to just move these fields at end of `mobj_t` so no need to add extra fields in `precipmobj_t`